vital/algo/detected_object_set_input.h

```diff
@@ -43,12 +46,14 @@ class VITAL_ALGO_EXPORT detected_object_set_input
   static std::string static_type_name() { return "detected_object_set_input"; }
 
   /// Open a file of detection sets.
+
   /**
    * This method opens a detection set file for reading.
    *
    * \param filename Name of file to open
    *
-   * \throws kwiver::vital::path_not_exists Thrown when the given path does not exist.
+   * \throws kwiver::vital::path_not_exists Thrown when the given path does not
+   * exist.
```

Comment:
So... Doxygen indentation is not well specified, but I do know this isn't right. Given it's also the wrong style (should be all `///`), do we even want to bother with this for the moment? (There are a bunch of spurious blank lines also being introduced due to the mix of `///` and `/**` that I'm ignoring for now.)

Reply:
I'll consider this fixed for now based on the fact that _some_ kind of indention was added.